app/controllers/embed_controller.rb

```diff
@@ -37,7 +37,7 @@ def topics
       end
     end
 
-    response.headers["X-Robots-Tag"] = "noindex, indexifembedded"
+    response.headers["X-Robots-Tag"] = "googlebot:noindex,indexifembedded"
```

Comment:
```suggestion
    response.headers["X-Robots-Tag"] = "noindex, googlebot:indexifembedded"
```

Actually I think `noindex` for all crawlers is what we want based on https://github.com/discourse/discourse/commit/9d5241d347fb1c42da6c557dc0ada86e5d31440a. I think we only want `indexifembedded` to be set for `googlebot`. 

@xfalcox Can you confirm this since you authored the change originally?